failing to fetch git-annex build from datalad/git-annex: bad credentials

```
0 upgraded, 0 newly installed, 0 to remove and 55 not upgraded.
Using curl as "curl --silent"
Getting artifacts_url from https://api.github.com/repos/datalad/git-annex/actions/workflows/build-ubuntu.yaml/runs?status=success&branch=master into /tmp/tmp.uomamAtfxT
Unable to get artifacts_url
{
  "message": "Bad credentials",
  "documentation_url": "https://docs.github.com/rest"
}
Error: Process completed with exit code 1.
```
see e.g.  https://github.com/datalad/datalad-extensions/runs/1571587991?check_suite_focus=true

@jwodder - do you have an immediate idea on what is wrong?

--- a/.github/workflows/test-datalad_container-maint.yaml
+++ b/.github/workflows/test-datalad_container-maint.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_container-maint.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '20 01 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_container-master.yaml
+++ b/.github/workflows/test-datalad_container-master.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_container-master.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '20 01 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_crawler-maint.yaml
+++ b/.github/workflows/test-datalad_crawler-maint.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_crawler-maint.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '00 02 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_crawler-master.yaml
+++ b/.github/workflows/test-datalad_crawler-master.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_crawler-master.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '00 02 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_hirni-maint.yaml
+++ b/.github/workflows/test-datalad_hirni-maint.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_hirni-maint.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '40 02 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_hirni-master.yaml
+++ b/.github/workflows/test-datalad_hirni-master.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_hirni-master.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '40 02 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_metalad-maint.yaml
+++ b/.github/workflows/test-datalad_metalad-maint.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_metalad-maint.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '40 01 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_metalad-master.yaml
+++ b/.github/workflows/test-datalad_metalad-master.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_metalad-master.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '40 01 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_neuroimaging-maint.yaml
+++ b/.github/workflows/test-datalad_neuroimaging-maint.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_neuroimaging-maint.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '00 01 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_neuroimaging-master.yaml
+++ b/.github/workflows/test-datalad_neuroimaging-master.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_neuroimaging-master.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '00 01 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_ukbiobank-maint.yaml
+++ b/.github/workflows/test-datalad_ukbiobank-maint.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_ukbiobank-maint.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '00 03 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_ukbiobank-master.yaml
+++ b/.github/workflows/test-datalad_ukbiobank-master.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_ukbiobank-master.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '00 03 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_webapp-maint.yaml
+++ b/.github/workflows/test-datalad_webapp-maint.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_webapp-maint.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '20 02 * * *'
 
 jobs:
   test:

--- a/.github/workflows/test-datalad_webapp-master.yaml
+++ b/.github/workflows/test-datalad_webapp-master.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-datalad_webapp-master.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '20 02 * * *'
 
 jobs:
   test:

--- a/extensions.yaml
+++ b/extensions.yaml
@@ -8,6 +8,7 @@
 #  apt_depends:  # dependencies to install via APT necessary for testing
 #  - package1
 #  - ...
+#  cron_time: <cron expression for scheduled runs>
 #
 # If no customizations necessary, just place "None" as the value
 #
@@ -22,18 +23,25 @@
 extensions:
  datalad_neuroimaging:
   apt_depends: [dcm2niix]
+  cron_time: '00 01 * * *'
  datalad_container:
   apt_depends: [singularity-container]
+  cron_time: '20 01 * * *'
  datalad_metalad:
   pypi: datalad-metalad
+  cron_time: '40 01 * * *'
  datalad_crawler:
+  cron_time: '00 02 * * *'
  datalad_webapp:
   pypi: datalad-webapp
   package: datalad_webapp
   tester: pytest
+  cron_time: '20 02 * * *'
  datalad_hirni:
   apt_depends: [dcm2niix, singularity-container]
   pypi: datalad-hirni
   package: datalad_hirni
   github: psychoinformatics-de/datalad-hirni
+  cron_time: '40 02 * * *'
  datalad_ukbiobank:
+  cron_time: '00 03 * * *'

--- a/templates/.github/workflows/test-{{extension.name}}-{{datalad_branch}}.yaml
+++ b/templates/.github/workflows/test-{{extension.name}}-{{datalad_branch}}.yaml
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/test-{{ extension.name }}-{{ datalad_branch }}.yaml'
   # all should be triggered on cron
   schedule:
-    - cron: '30 01 * * *'
+    - cron: '{{ extension.cron_time }}'
 
 jobs:
   test:

